README.md

```diff
@@ -48,28 +49,3 @@ options:
   - "--exclude: '*'",
   - "--include: 'results/*'"
 ```
-
-## Development
-
-Requires [Docker](https://www.docker.com/).
-
-### Building, Uploading, and Using the Docker Image
-1. Download [Docker Toolbox](https://www.docker.com/products/docker-toolbox).
```

Comment:
Yep, was glad you added it! But now even better that we've cut out that entire process :zap: